pkg/util/kube.go

```diff
@@ -18,3 +30,62 @@ const (
 	// Kube annotation for podman volume image.
 	VolumeImageAnnotation = "volume.podman.io/image"
 )
+
+// ExtractPlayReader provide an io.Reader given a http.Request object
+// the function will extract the Content-Type header, if not provided, the body will be returned
+// of the header define a text format (json, yaml or text) it will also return the body
+// if the Content-Type is tar, we extract the content to the anchorDir and try to read the `play.yaml` file
+func ExtractPlayReader(anchorDir string, r *http.Request) (io.Reader, error) {
```

Comment:
I would rather have this logic in pkg/api/handlers/libpod/kube.go as it is API endpoint specific and not some random generic util that would be useful elsewhere

Reply:
I moved it to `pkg/api/handlers/libpod/kube.go`

Reply:
I moved it to `pkg/api/handlers/libpod/kube.go`